e2e/app/page/notebook-page.ts

```diff
@@ -24,7 +24,7 @@ enum CssSelector {
 enum Xpath {
   fileMenuDropdown = '//a[text()="File"]',
   downloadMenuDropdown = '//a[text()="Download as"]',
-  downloadIpynbButton = '//*[@id="download_ipynb"]/a',
+  downloadIpynbButton = '//*[@id="download_script"]/a',
```

Comment:
notebook image updated. Fix broken xpath selector.

Reply:
Sorry - thought I'd run puppeteer locally to check for this kind of issue, but it's possible it was failing for other reasons at the time, I can't remember now.